fix(howto): add dashboard tab to publish instance howto page [WD-30809] (#513)
canonical/anbox-cloud-docs: howto/images/publish-instance-as-image.md

    List of devices attached
    emulator-5558   device

:::

:::{tab-item} Dashboard
:sync: dashboard

Navigate to the *Instances* page and create and start an instance from an existing image using the *Create instance* button.
Alternatively, you can also navigate to the *Images* page, find the image you want to use as a base, and click the *Create instance* button ( ![create instance icon](/images/icons/create-instance-icon.png) ) either in the *Images* page or the *Image details* page.

Wait until the instance becomes *running*, then click on the instance name to open the *Instance details* page. From there, open the *Terminal* tab and make any modifications you want to include in the new image (e.g. install a package, create a file).

> When you're done, you can navigate back to the *Instances* page from the side navigation. Here, you can click on the *Publish* button ( ![publish instance icon](/images/icons/publish-instance-icon.png) ) to open a dialog where you can customize the image name, and finally confirm publishing the instance by clicking on the *Publish* button.

Reviewer: ```suggestion
When you're done, publish( ![publish instance icon](/images/icons/publish-instance-icon.png) )  the instance from the *Instances* page. When publishing, you will be able to customize the image name.
```
